src/Components/CollectionsHubRails/ArtistSeriesRail/ArtistSeriesEntity.tsx

```diff
@@ -24,9 +25,24 @@ export const ArtistSeriesEntity: React.FC<ArtistSeriesEntityProps> = ({
   const bgImages = hits.map(hit => hit.image.url)
   const imageSize =
     bgImages.length === 1 ? 265 : bgImages.length === 2 ? 131 : 85
+  const { trackEvent } = useTracking()
+  const onClickLink = event => {
```

Comment:
So that this is a bit more readable, can a) a line break be added between 28 and 29 so that the function handler jumps out; and b) the variable renamed to `handleLinkClick`; `onLinkClick` is the form typically passed down from props e.g., 

```jsx
<Foo onClick={...}>
```
and then inside 
```jsx
const Foo = props => {
  if (props.onClick) { 
    ...
  }
}
```
Since we're "handling" an action, these functions are typically prefixed with "handle"; so :

```jsx
const handleLinkClick = event => { ... }
```

(Similarly, if it were a mouse-over, it would be `handleMouseOver`, etc.)

(cc @xtina-starr )